lib/spack/spack/build_systems/cmake.py

```diff
@@ -142,8 +172,8 @@ def _run_default_function(self):
             tty.msg('Skipping default build sanity checks [method `check` not implemented]')  # NOQA: ignore=E501
 
     def check(self):
-        """Default test: search the Makefile for the target ``test``
-        and run them if found.
+        """Searches the Makefile for the target ``test`` and runs
+        it if found.
```

Comment:
Try: "Searches the CMake-generated Makefile for the target ``test`` and runs
+        it if found."